tools/dds/dds-server/lrs-device-controller.cpp

```diff
@@ -500,7 +501,16 @@ lrs_device_controller::lrs_device_controller( rs2::device dev, std::shared_ptr<
                 auto & server = it->second;
                 if( _bridge.is_streaming( server ) )
                 {
-                    server->publish( static_cast< const uint8_t * >( f.get_data() ), f.get_data_size(), f.get_frame_number() );
+                    realdds::topics::image_msg image;
+                    auto data = static_cast< const uint8_t * >( f.get_data() );
+                    image.raw_data.assign( data, data + f.get_data_size() );
```

Comment:
Can we move the data?
Maybe construct a new vector on this range and move to `raw_data`

Reply:
Constructing a new vector (meaning a copy) and moving it is the same as assigning the data. We assigned the data inside `publish` before, too, so this is not a change.